src/Fixer/CastNotation/ShortScalarCastFixer.php

```diff
@@ -60,8 +54,20 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
-    protected function getDescription()
+    public function getDefinition()
     {
-        return 'Cast "(boolean)" and "(integer)" should be written as "(bool)" and "(int)". "(double)" and "(real)" as "(float)".';
+        return new FixerDefinition(
+            'Cast "(boolean)" and "(integer)" should be written as "(bool)" and "(int)". "(double)" and "(real)" as "(float)".',
+            null,
+            array(new CodeSample("<?php\n\$a = (boolean) \$b; \$a = (integer) \$b; \$a = (boolean) \$b; \$a = (double) \$b;"))
```

Comment:
duplicated `boolean`, missing `float`.
i would split them to be followed line by line